src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -88,9 +88,16 @@ internal static void UpdateTextHtml(this UILabel platformLabel, ILabel label)
 			};
 
 			NSError nsError = new();
-#pragma warning disable CS8601
-			platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
-#pragma warning restore CS8601
+			CoreFoundation.DispatchQueue.MainQueue.DispatchAsync(() =>
+			{
+				platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
+				if (label.Handler is ILabelHandler labelHandler)
+				{
+					labelHandler.UpdateValue(nameof(ILabel.TextColor));
+					labelHandler.UpdateValue(nameof(ILabel.Font));
+				}
+				label.InvalidateMeasure();
```

Comment:
What happens if we don't include this?

Reply:
Without `label.InvalidateMeasure();` the label doesn't render